src/ffi/rust.rs

```diff
@@ -136,7 +136,8 @@ impl From<FlushCompress> for MZFlush {
     fn from(value: FlushCompress) -> Self {
         match value {
             FlushCompress::None => Self::None,
-            FlushCompress::Partial | FlushCompress::Sync => Self::Sync,
+            FlushCompress::Partial => Self::Partial,
```

Comment:
@oyvindln Do you know if there was a reason *not* to use partial flushes before? If not, then it's probably safe to map it 1:1, but I wanted to check with you.
Thanks for your help!

Reply:
I have no idea

Reply:
I looked into it by digging through the history, and it's unclear why things are the way they are.

There was a time when there was direct conversion.
```Rust
        flush: FlushCompress,
    ) -> Result<Status, CompressError> {
        let flush = MZFlush::new(flush as i32).unwrap();
```

The revision that actually introduced this line didn't mention why either.

So I think it's fair to merge this.